File: ProggramingFundamentalsAndUnitTesting/WhileLoopsLab/SumDigits.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace WhileLoopsLab
{
    internal class SumDigits
    {
        static void Main_(string[] args)
        {
            string str = Console.ReadLine();
            
            string[] elements = str.ToCharArray().Select(c => c.ToString()).ToArray();

            

            int sum = elements.Select(int.Parse).Sum();
            Console.WriteLine(sum);

        }
    }
}

File: Back-EndTechnologiesBasics/JavaScriptBasics/sumDigits.js
function SumDigits(input){
    textInput = input.toString();
    let sum = 0;
    for (let i = 0; i < textInput.length; i++) {
        let digit = Number(textInput[i]);
        sum += digit;
    }
    console.log(sum); 
}
SumDigits(245678);